packages/datagateway-common/src/state/actions/actions.types.tsx

```diff
@@ -0,0 +1,274 @@
+  'datagateway_common:fetch_instrument_details_failure';
+export const FetchInstrumentDetailsSuccessType =
+  'datagateway_common:fetch_instrument_details_success';
+
+export const FetchInstrumentCountRequestType =
+  'datagateway_common:fetch_instrument_count_request';
+export const FetchInstrumentCountFailureType =
+  'datagateway_common:fetch_instrument_count_failure';
+export const FetchInstrumentCountSuccessType =
+  'datagateway_common:fetch_instrument_count_success';
+
+export const FetchFacilityCyclesRequestType =
+  'datagateway_common:fetch_facility_cycles_request';
+export const FetchFacilityCyclesFailureType =
+  'datagateway_common:fetch_facility_cycles_failure';
+export const FetchFacilityCyclesSuccessType =
+  'datagateway_common:fetch_facility_cycles_success';
+
+export const FetchFacilityCycleCountRequestType =
+  'datagateway_common:fetch_facility_cycle_count_request';
+export const FetchFacilityCycleCountFailureType =
+  'datagateway_common:fetch_facility_cycle_count_failure';
+export const FetchFacilityCycleCountSuccessType =
+  'datagateway_common:fetch_facility_cycle_count_success';
+
+export const FetchDownloadCartRequestType =
+  'datagateway_common:fetch_download_cart_request';
+export const FetchDownloadCartFailureType =
+  'datagateway_common:fetch_download_cart_failure';
+export const FetchDownloadCartSuccessType =
+  'datagateway_common:fetch_download_cart_success';
+
+export const AddToCartRequestType = 'datagateway_common:add_to_cart_request';
+export const AddToCartFailureType = 'datagateway_common:add_to_cart_failure';
+export const AddToCartSuccessType = 'datagateway_common:add_to_cart_success';
+
+export const RemoveFromCartRequestType =
+  'datagateway_common:remove_from_cart_request';
+export const RemoveFromCartFailureType =
+  'datagateway_common:remove_from_cart_failure';
+export const RemoveFromCartSuccessType =
+  'datagateway_common:remove_from_cart_success';
+
+export const FetchAllIdsRequestType =
+  'datagateway_common:fetch_all_ids_request';
+export const FetchAllIdsFailureType =
+  'datagateway_common:fetch_all_ids_failure';
+export const FetchAllIdsSuccessType =
+  'datagateway_common:fetch_all_ids_success';
+
+export interface SortTablePayload {
+  column: string;
+  order: Order | null;
+}
+
+export interface FilterTablePayload {
+  column: string;
+  filter: Filter | null;
+}
+
+export interface ConfigureStringsPayload {
+  res: ApplicationStrings;
+}
+
+export interface ConfigureFacilityNamePayload {
+  facilityName: string;
+}
+
+export interface FeatureSwitchesPayload {
+  switches: FeatureSwitches;
+}
+
+// eslint-disable-next-line @typescript-eslint/no-empty-interface
+export interface FeatureSwitches {}
+
+export interface ConfigureUrlsPayload {
+  urls: URLs;
+}
+
+export interface URLs {
+  idsUrl: string;
+  apiUrl: string;
+  downloadApiUrl: string;
+}
+
+export interface ConfigureBreadcrumbSettingsPayload {
+  settings: BreadcrumbSettings;
+}
+
+export interface BreadcrumbSettings {
```

Comment:
Can we delete interfaces that aren't supposed to be defined in common? aka any that are in table shouldn't be here